features/child_workflow/signal/feature.go

```diff
@@ -0,0 +1,52 @@
+package signal
+
+import (
+	"context"
+	"time"
+
+	"go.temporal.io/features/harness/go/harness"
+	"go.temporal.io/sdk/client"
+	"go.temporal.io/sdk/workflow"
+)
+
+const (
+	UnblockSignal  = "unblock-signal"
+	UnblockMessage = "unblock"
+)
+
+// A workflow that starts a child workflow, unblocks it, and returns the result
+// of the child workflow.
+func Workflow(ctx workflow.Context) (string, error) {
+	ctx = workflow.WithChildOptions(ctx, workflow.ChildWorkflowOptions{
+		WorkflowExecutionTimeout: 10 * time.Minute,
+		WorkflowTaskTimeout:      time.Minute,
+	})
+	childWorkflowFut := workflow.ExecuteChildWorkflow(ctx, ChildWorkflow)
+	childWorkflowFut.SignalChildWorkflow(ctx, UnblockSignal, UnblockMessage)
+	result := ""
+	err := childWorkflowFut.Get(ctx, &result)
+	if err != nil {
+		return "", err
+	}
+	return result, nil
```

Comment:
```suggestion
	var result string
	err := childWorkflowFut.Get(ctx, &result)
	return result, err
```

Subjective, no need to change anything

Reply:
I like the visually cleaner style of that change. I don't have much Go experience and have a question: so, if there's an error, then it's possible that the data written to `&result` is corrupt or partially written, etc. Is it considered good Go philosophy to say "There was an error. I'm giving you your requested data as well, but all bets are off regarding it -- e.g. it may not even be a valid instance of its type".

Reply:
E.g. languages like Rust with result ADTs don't do that, but in languages with exceptions the user can provide a string representation of the result.

Reply:
It's a good question. Often it is implied in Go that "if there are two return types and the second is an error, they are mutually exclusive and err != nil means result has no value unless documented otherwise". But that's not a hardfast rule in all of Go. But I can say result and error are mutually exclusive for Temporal workflows, so a non-nil error discards results.

Reply:
OK thanks for discussing! I think I'll leave this as-is for now and keep the issue in mind as I get more Go experience. So I'm going to provisionally say that I'm going to avoid doing it in case it gives a reader the impression that corrupt data might be returned.